samples/functions/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@google-cloud/bigtable-samples",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": "googleapis/nodejs-bigtable",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "test": "mocha test/*.test.js --timeout=20000",
+    "start": "functions-framework --target=get"
+  },
+  "dependencies": {
+    "@google-cloud/bigtable": "^2.3.2",
```

Comment:
Could `@google-cloud/functions-framework` be a development dependency? I'm wondering if we can get away with just showing the structure of the cloud function to the customer, without pulling in this dep (except for testing).